modules/stereo_corresp/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+set(the_description "Stereo Correspondance Algorithms")
```

Comment:
since this is the first commit, let's rename the module from stereo_corresp to just "stereo".